src/index.ts

```diff
@@ -0,0 +1,96 @@
+import express, { Express } from "express";
+import {
+  Action,
+  useExpressServer,
+  UnauthorizedError,
+  BadRequestError
+} from "routing-controllers";
+import { BotController } from "./controllers";
+import dotenv from "dotenv";
+import { decode } from "jsonwebtoken";
+import * as bodyParser from "body-parser";
+import * as winston from "winston";
+import * as expressWinston from "express-winston";
+import "reflect-metadata";
+import { IJwt } from "./models/jwt";
+import { SecurityMiddleware } from "./middlewares/SecurityMiddleware";
+import configApp from "./configs/app";
+dotenv.config();
+
+const app: Express = express();
+
+// Log the HTTP requests
+app.use(
+  expressWinston.logger({
+    transports: [new winston.transports.Console()],
+    format: winston.format.combine(
+      winston.format.json(),
+      winston.format.colorize()
+    )
+  })
+);
+
+useExpressServer(app, {
+  authorizationChecker: async (action: Action) => {
```

Comment:
Can you import the authorization checker from a module?

Reply:
sure